lib/mix/lib/mix/scm/git.ex

```diff
@@ -52,11 +52,12 @@ defmodule Mix.SCM.Git do
         File.cd!(opts[:dest], fn ->
           rev_info = get_rev_info
```

Comment:
Can we get the tag information here? I would like to avoid multiple calls into git.